src/main/java/io/axoniq/axonserver/connector/event/impl/BufferedAggregateEventStream.java

```diff
@@ -60,6 +66,9 @@ public Event next() throws InterruptedException {
         } else {
             taken = take();
         }
+        if (taken != null) {
+            lastSequenceNumber = taken.getAggregateSequenceNumber();
```

Comment:
Just for certainty, but the `lastSequenceNumber` is added only for logging purposes right now, right?

Reply:
Yes, that is correct. It's important for analyzing the problem if it occurs

Reply:
Gotcha, then we leave it as is.